e2e/pnpm_workspace/app/a/package.json

```diff
@@ -0,0 +1,8 @@
+{
+    "name": "@app/a",
+    "private": true,
+    "dependencies": {
+        "@aspect-test/g": "1.0.0",
+        "@lib/a": "workspace:*"
```

Comment:
i guess the value on this line is a pnpm-specific feature?

Reply:
🌮